pubtools/_quay/quay_client.py

```diff
@@ -109,19 +111,20 @@ def get_manifest(self, image, raw=False, manifest_list=False, v2s1_manifest=Fals
         else:
             return response.json()
 
-    def get_manifest_digest(self, image, v2s1_manifest=False):
+    def get_manifest_digest(self, image, media_type=None):
         """
         Get manifest of the specified image and calculate its digest by hashing it.
 
         Args:
             image (str):
                 Image address for which to calculate the digest.
-            v2s1_manifest (bool):
-                Whether to calculate digest of image's V2S1 manifest.
+            media_type (str):
+                Type of manifest Can be application/vnd.docker.distribution.manifest.v2+json
```

Comment:
Is there a need to restrict ML? Also, there's no actual check performed about allowed types